Ginger/GingerCore/GingerOCR/GingerOcrOperations.cs

```diff
@@ -353,19 +353,26 @@ private static void ReadTextBetweenTwoLabels(string firstLabel, string secondLab
                             string lineTxt = iter.GetText(PageIteratorLevel.TextLine);
                             if (lineTxt.Contains(firstLabel))
                             {
-                                firstIndexOf = lineTxt.IndexOf(firstLabel) + firstLabel.Length;
+                                firstIndexOf = lineTxt.IndexOf(firstLabel);
+                                int startConcatIndex = lineTxt.IndexOf(firstLabel) + firstLabel.Length;
```

Comment:
Index of firstlabel is already calculated and stored in firstIndexOf variable, use the same variable here